### ISSUE TITLE
comma causes problem at custom function parameters at excel

### ISSUE THREAD

**Issue by emindeniz99** [OfficeDev/office-js-docs-pr#1784]
It gives this error https://support.office.com/en-us/article/how-to-avoid-broken-formulas-8309381d-33e8-42f6-b889-84ef6df1d586

"CONTOSO.GETSTARCOUNT("OfficeDev"**,** "Excel-Custom-Functions")"
When using semi colon instead of comma, this problem is solved.
"CONTOSO.GETSTARCOUNT("OfficeDev"   **;**   "Excel-Custom-Functions")"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a30fb645-c853-a93a-4137-1e2bf7398a11
* Version Independent ID: 75fb99ac-72e8-7bdc-ff1b-85e7ed930d50
* Content: [Excel custom functions tutorial - Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/tutorials/excel-tutorial-create-custom-functions?tabs=excel-windows)
* Content Source: [docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/tutorials/excel-tutorial-create-custom-functions.md)
* Product: **excel**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by AlexJerabek** [comment]
Thanks for reporting this @emindeniz99. 

@mscharlock, could you please investigate?

**Comment by mscharlock** [comment]
Hi @emindeniz99 - Thanks for writing in with this issue. I can't reproduce it. 
Can I ask what version of Excel you are using and on what platform (web, mac, windows) and if you are part of the Office Insider program? 
Thanks!

**Comment by emindeniz99** [comment]
![image](https://user-images.githubusercontent.com/44745432/80135657-28481000-85a9-11ea-995c-72dc5066937f.png)
![image](https://user-images.githubusercontent.com/44745432/80135789-68a78e00-85a9-11ea-9b89-3f9c1e1f1afe.png)
![image](https://user-images.githubusercontent.com/44745432/80135840-807f1200-85a9-11ea-94fe-837c977bd7e6.png)

I tested on those platforms
Excel Online 16.0.12821.35900
Office 365 MSO Excel 16.0.12624.20424 64-bit Windows 10 Home 1909

I suspected from language settings, I changed excel lang to eng. I couldn't change the online versions.
My OS Language is Turkish. 

While I was writing this comment, I researched it again and I found solution, https://answers.microsoft.com/tr-tr/msoffice/forum/all/merhaba-microsoft-excel-for-mac-2011-excel/82e6f1e5-70f7-4b60-bbd2-c30bd3c6ddd1
TR-ENG default separator causes the problem.
Thank you very much for your attention
